packages/bugs/package.json

```diff
@@ -1,4 +1,5 @@
 {
+  "name": "bugs",
```

Comment:
Had errors with using `pnpm install` at repo root... so used bun but go an error that said bugs package didn't have a name, so just added it

Reply:
oh interesting, thanks for adding that